doc/classes/ArrayMesh.xml

```diff
@@ -3,6 +3,23 @@
 	<brief_description>
 	</brief_description>
 	<description>
+		The [code]ArrayMesh[/code] is used to construct a [Mesh] by specifying the attributes as arrays. The most basic example is the creation of a single triangle
+		[codeblock]
+		var vertices = PoolVector3Array()
+		vertices.push_back(Vector3(0,1,0))
+		vertices.push_back(Vector3(1,0,0))
+		vertices.push_back(Vector3(0,0,1))
+		# Initialize the ArrayMesh.
+		var arr_mesh = ArrayMesh.new()
+		var arrays = Array()
```

Comment:
Not sure if you need to use `Array()` here. `[]` in place of `Array()` probably works just fine.

Reply:
var arrays = Array() works too but I agree using [] is better. This code happens to be a conversion from a GDN C++ module I wrote and I did a semi-literal conversion.